tests/FSharp.Compiler.ComponentTests/Interop/SimpleInteropTests.fs

```diff
@@ -5,7 +5,7 @@ namespace FSharp.Compiler.ComponentTests.Interop
 open Xunit
 open FSharp.Test.Utilities.Compiler
 
-module ``C# <-> F# basic interop`` =
+module ``Simple interop verification`` =
```

Comment:
For some reason, VS Test Explorer wouldn't show these tests under the "Interop" category, with 'C#' and 'F#' in the module name.

Reply:
Seems like a test explorer bug. @shyamnamboodiripad ?

Reply:
Looks like it is, so before, when the name was ` ``C# <-> F# basic interop`` `, the tree was
```
FSharp.Compiler.ComponentTests
∟    Interop
     ∟    <Interop tests from 'C# <-> F# basic interop' module>
FSharp.Compiler.ComponentTests.Interop
∟    <Rest of interop test suites>
```

As oppose to 
```
FSharp.Compiler.ComponentTests.Interop
∟    C# <-> F# basic interop
∟    <Rest of interop test suites> 

```

This rename fixed it.